frontend/src/apiMocks/handlers.ts

```diff
@@ -3,11 +3,11 @@ import { CustomAliasData, RandomAliasData } from "../hooks/api/aliases";
 import { ProfileData } from "../hooks/api/profile";
 import {
   mockIds,
-  domainaddresses,
-  profiles,
-  relayaddresses,
-  runtimeData,
-  users,
+  mockedDomainaddresses,
+  mockedProfiles,
+  mockedRlayaddresses,
```

Comment:
suggetion (non-blocking): Spelling 

Note: this would apply to the whole PR. 

```suggestion
  mockedRelayaddresses,
```

Reply:
Haha nice catch! 71a6d438